packages/shared/src/components/squads/layout/useSquadDirectoryLayout.ts

```diff
@@ -26,7 +26,7 @@ export const useSquadDirectoryLayout = (): SquadDirectoryLayoutReturn => {
     hasSquad,
     squads,
     mySquadsTab: {
-      isVisible: hasSquad && !isLaptop,
```

Comment:
This should still be the same condition, right?

Reply:
In order to see the additional tab you need to have some squad and navigate from mobile/tablet. Something is wrong?